nautobot/extras/tables.py

```diff
@@ -421,3 +422,27 @@ class RelationshipAssociationTable(BaseTable):
     class Meta(BaseTable.Meta):
         model = RelationshipAssociation
         fields = ("relationship", "source", "destination", "actions")
+
+
+class ComputedFieldTable(BaseTable):
+    pk = ToggleColumn()
+    label = tables.Column(linkify=True)
+
+    class Meta(BaseTable.Meta):
+        model = ComputedField
+        fields = (
+            "pk",
+            "label",
+            "slug",
+            "content_type",
+            "description",
+            "template",
```

Comment:
I held the same suspicion, which is why I made it optional "just in case", but giving it a second thought, am more that happy to remove this as an option. Will do.